### ISSUE TITLE
[Feature] Add Prow as community bot in KubeVela

### ISSUE THREAD

**Issue by FogDong** [kubevela/kubevela#5815]
**Is your feature request related to a problem? Please describe.**

Prow bot is popular in the cloud-native community, it would be nice if we can integrate it to the KubeVela community and use it for issue labeling, reviewing requests, etc.

**Describe the solution you'd like**

**Describe alternatives you've considered**

**Additional context**

**Comment by sbdtu5498** [comment]
@FogDong what sort of feature do you want to see via prow for now? It is worth noting that test infra is actively working on integrating GitHub actions workflow with prow. So in the future, we can directly leverage GitHub workflows scripts for testing via prow. But for now, what are the required use cases? Like do we want something like Slack infra or what? An elaborate explanation would be appreciated.

**Comment by FogDong** [comment]
We'd like to replace kubevela-bot with prow. In this way, we can use the `/label`, `/retest` capabilities, with kubevela-bot, we can only use it for `/assign`, so it's better to use a mature bot like prow.

**Comment by sbdtu5498** [comment]
Sure that would be great. I would like to work on this one.

**Comment by Somefive** [comment]
@sbdtu5498 Prow needs extra machine to deploy and there is also issues for exposing a url for github to call webhook. Therefore, for now we can focus on some simple command.

#5936 gives a example of how to add support for "/retest" command. Maybe you can follow it and try to develop the "/label" command.

**Comment by sbdtu5498** [comment]
Yes, Prow requires a host Kubernetes cluster to work. So are you currently planning to stick to the GitHub actions or migrate stuff to Prow? and regarding #5936, sure thing, I would follow the work and write actions for /label accordingly.

**Comment by Somefive** [comment]
Yes. Currently we do not find the resource to how the Prow, especially a DNS for calling webhook. So I would recommend to use the GitHub action until we got other more resources.

**Comment by briankane** [comment]
This issue has been inactive for over 2 years and is scheduled for closure as part of our backlog maintenance.

  **If this issue is still relevant:**
  - Please comment below to keep it open
  - Provide updated context if the original description is outdated

  **Timeline:** If there's no activity by **Friday, October 24th**, this issue will be automatically closed.

  **Note:** Issues can always be reopened if they become relevant again. We encourage the community to reopen and reprioritize any incorrectly 
  closed issues.

**Comment by briankane** [comment]
This issue has been closed following the stale issue notice. 

If this issue is still relevant and should remain open, please:
1. Reopen the issue
2. Remove the 'stale' label
3. Provide an update on the current status or importance

Thank you for your understanding and contribution to KubeVela!
